### ISSUE TITLE
Exclude fleets information systematically

### ISSUE THREAD

**Issue by kellijohnson-NOAA** [nwfsc-assess/nwfscDiag#15]
Automate the exclusion of entire datasets, which could serve as both a sensitivity and for the exploration of retrospective patterns.

_Originally posted by @brianlangseth-NOAA in https://github.com/nwfsc-assess/nwfscDiag/issues/12

**Comment by brianlangseth-NOAA** [comment]
@kellijohnson-NOAA Yes, I think the most straightforward way would be to specify a fleet number or name and have everything removed from that (catch, index, comps). A systematic approach could work too where a function is run and all fleets are removed one by one, but  I would prefer a function that does each fleet on its one. Its easy to run that function for each fleet, and in many cases we may only need it for one or two fleets anyway.
